data/model/autoprune.py

```diff
@@ -128,6 +195,22 @@ def get_namespace_autoprune_policy(orgname, uuid):
         return None
 
 
+def get_repository_autoprune_policy_by_uuid(repo_name, uuid):
+    """
+    Get the specific autopruning policy for the specified repository by uuid.
+    """
+    try:
+        row = (
+            RepositoryAutoPrunePolicyTable.select(RepositoryAutoPrunePolicyTable)
+            .join(Repository)
+            .where(Repository.name == repo_name, RepositoryAutoPrunePolicyTable.uuid == uuid)
```

Comment:
`uuid` should be universally unique, i don't think adding the repo_name adds anything

Reply:
agreed. I added it similar to [get_namespace_autoprune_policy(orgname, uuid)](https://github.com/quay/quay/blob/master/data/model/autoprune.py#L115). I can follow up with a separate PR to clean up both as there are tests using it too.